src/tasks/AutoFountainTask.py

```
@@ -0,0 +1,258 @@
+        self.name = "自动喷泉签到"
+        self.icon = FluentIcon.SYNC
+        self.group_name = "都市闲趣"
+        self.group_icon = FluentIcon.ACCEPT
+
+    def run(self):
+        super().run()
+        try:
+            self.do_run()
+        except TaskDisabledException:
+            raise
+        except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)